PocketKit/Sources/Sync/Source.swift

```diff
@@ -31,7 +31,7 @@ public protocol Source {
 
     func archive(recommendation: Slate.Recommendation)
 
-    func fetchArchivedItems() async throws -> [ArchivedItem]
+    func fetchArchivedItems(favorited: Bool) async throws -> [ArchivedItem]
```

Comment:
Two thoughts here:

1. How about renaming `favorited` to `isFavorite` to stay consistent
2. How about making the `Bool` argument a `Bool?`. That way we'd be able to *theoretically* request items that are not favorited. Then when we ultimately build the `SavedItemsFilter`, we would only set the `isFavorite` argument if it is not nil.